Easy_Problems/Coderbyte3.js

```diff
@@ -0,0 +1,11 @@
+function LongestWord(sen) {
```

Comment:
Since you are already looping through the list once why not just check for the longest word in there instead of sorting it afterwords

```
function LongestWord(sen) {
	var array = sen.split(" ");
  var longest = array[0].replace(/\W/g, "");

	for (var i = 1; i < array.length; i++) {
		var currentWord = array[i].replace(/\W/g, "");

    if (currentWord.length > longest.length) {
      longest = currentWord;
    }
	}

  return longest;
};

console.log(LongestWord("this is anna"));
```